tests/phpunit/elementor/modules/atomic-widgets/elements/test-has-template.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Elementor\Testing\Modules\AtomicWidgets\Elements;
+
+use Elementor\Plugin;
+use ElementorEditorTesting\Elementor_Test_Base;
+use Spatie\Snapshots\MatchesSnapshots;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Test_Has_Template extends Elementor_Test_Base {
+	use MatchesSnapshots;
+
+	/**
+	 * @dataProvider element_data_provider
+	 */
+	public function test_render(array $element_data):void {
```

Comment:
Some more linting OCD
```suggestion
	public function test_render(array $element_data): void {
```